pyproject.toml

```diff
@@ -3,7 +3,7 @@ line-length = 100
 
 [tool.poetry]
 name = "dbt-copilot-tools"
-version = "2.0.2"
+version = "2.0.3"
```

Comment:
Not sure this change/refactor requires a version upgrade.

Reply:
I think I had to do this to get past the pre-commit hooks. I just tried to reverse this and it bumped up the version to 3.0.1

Reply:
`git commit --no-verify` for today.

Then we maybe need a ticket to not bump the version if the only thing that has changed is the test files.